From ebc5a09db457ee1be9996711463cbbafe5ea72d5 Mon Sep 17 00:00:00 2001
From: James Cowgill <james410@cowgill.org.uk>
Date: Mon, 13 Apr 2015 16:13:53 +0100
Subject: [PATCH] Fix build failure with GCC 5
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The error is:
clitest.c: In function ‘cmd_test’:
clitest.c:75:45: error: ISO C does not support '__FUNCTION__' predefined identifier [-Werror=pedantic]
     cli_print(cli, "called %s with \"%s\"", __FUNCTION__, command);
                                             ^
Fixed by using __func__ from C99.
---
 clitest.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clitest.c b/clitest.c
index 09552c4..ace87bc 100644
--- a/clitest.c
+++ b/clitest.c
@@ -72,7 +72,7 @@ int winsock_init()
 int cmd_test(struct cli_def *cli, const char *command, char *argv[], int argc)
 {
     int i;
-    cli_print(cli, "called %s with \"%s\"", __FUNCTION__, command);
+    cli_print(cli, "called %s with \"%s\"", __func__, command);
     cli_print(cli, "%d arguments:", argc);
     for (i = 0; i < argc; i++)
         cli_print(cli, "        %s", argv[i]);
-- 
2.17.0

